### ISSUE TITLE
CUDA_ERROR_OUT_OF_MEMORY on a 1080ti

### ISSUE THREAD

**Issue by notbrian** [lengstrom/fast-style-transfer#160]
Anyone have any success with this repo on Windows 10 using Tensorflow 1.8.0 and Python 2.7.9? I'm forced to use these versions because TF 0.11.0 isn't supported on Windows as far as I can tell. 

When I run style.py no matter the sizes of the style image, testing images or batch size I get a `CUDA_ERROR_OUT_OF_MEMORY` error.

`Train set has been trimmed slightly..
(1, 40, 54, 3)
2018-07-08 20:03:26.816556: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1356] Found device 0 with properties: 
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.607
pciBusID: 0000:01:00.0
totalMemory: 11.00GiB freeMemory: 9.10GiB
2018-07-08 20:03:26.816781: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1435] Adding visible gpu devices: 0
2018-07-08 20:03:27.871478: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:923] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-07-08 20:03:27.871663: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:929]      0
2018-07-08 20:03:27.874074: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:942] 0:   N
2018-07-08 20:03:27.874927: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1053] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 8805 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:01:00.0, compute capability: 6.1)
2018-07-08 20:03:28.354031: E T:\src\github\tensorflow\tensorflow\stream_executor\cuda\cuda_driver.cc:936] failed to allocate 8.60G (9233113856 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY
2018-07-08 20:03:28.767738: E T:\src\github\tensorflow\tensorflow\stream_executor\cuda\cuda_driver.cc:936] failed to allocate 7.74G (8309802496 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY
2018-07-08 20:03:29.215930: E T:\src\github\tensorflow\tensorflow\stream_executor\cuda\cuda_driver.cc:936] failed to allocate 6.96G (7478821888 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY
2018-07-08 20:03:29.652668: E T:\src\github\tensorflow\tensorflow\stream_executor\cuda\cuda_driver.cc:936] failed to allocate 6.27G (6730939392 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY
`

**Comment by notbrian** [comment]
Managed to get it working with `config.gpu_options.per_process_gpu_memory_fraction` or `config.gpu_options.allow_growth = True` from https://www.tensorflow.org/programmers_guide/using_gpu#allowing_gpu_memory_growth

However, the GPU usage is super low 0-5%, which makes me suspect it isn't using the GPU much

**Comment by isdzaurov** [comment]
Hello Brian. I have this error to. Please tell me where need to paste config.gpu_options.allow_growth = True ?

**Comment by leefeel** [comment]
Hey @notbrian! I would like to use this project on Windows 10. Were you able to make it run on your GPU? Thank you!

**Comment by notbrian** [comment]
> Hey @notbrian! I would like to use this project on Windows 10. Were you able to make it run on your GPU? Thank you!

Yes, I did get it to run and extract different styles. But it's been almost a year since I last touched it.

**Comment by FantasyJXF** [comment]
> > Hey @notbrian! I would like to use this project on Windows 10. Were you able to make it run on your GPU? Thank you!
> 
> Yes, I did get it to run and extract different styles. But it's been almost a year since I last touched it.

Hi, I met the problem too, and I tried the `config.gpu_options.per_process_gpu_memory_fraction` or `config.gpu_options.allow_growth = True` both, in my test, the inference procedure tooks 20G graphic memory from my total 22G memory
